exonum/src/messages/types.rs

```diff
@@ -40,6 +40,24 @@ pub struct SignedMessage {
     pub author: PublicKey,
     /// Digital signature over `payload` created with `SecretKey` of the author of the message.
     pub signature: Signature,
+
+    /// No-op field for forward compatibility.
+    #[protobuf_convert(skip)]
```

Comment:
I don't think extending `SignedMessage` is feasible in 1.x; it's a simple type encompassing data signed with a Ed25519 key.